packages/2019-transportation/src/components/DisturbanceStops/index.js

```diff
@@ -1,46 +1,82 @@
-import React, { useEffect } from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
```

Comment:
All data requests should be done in the state directory. If you run `yarn card` from the root of the repo, it will build the whole card structure for you so all you have to do is add your URL and sub in your visualization and card info. You should be able to copy paste your stuff from these cards

Reply:
Nevermind. I understand you have a special use case

Reply:
yeah i started with axios/hooks to do a quick and dirty paging/params solution. the API endpoints seem to have a little more power behind them now, so i think it should be good to migrate from axios/hooks back into reduxful